client/src/views/users/UserDetailsModal.vue

```diff
@@ -0,0 +1,206 @@
+        </ion-text>
+      </div>
+      <div>
+        <ion-text class="text4">
+          {{ $t('UsersPage.UserDetailsModal.subtitles.commonWorkspaces') }}
+        </ion-text>
+      </div>
+      <div>
+        <ion-chip
+          v-if="user.isRevoked()"
+          color="danger"
+        >
+          <ion-icon
+            :icon="ellipse"
+            class="revoked"
+          />
+          <ion-label>
+            {{ $t('UsersPage.UserDetailsModal.subtitles.revoked') }}
+          </ion-label>
+        </ion-chip>
+      </div>
+      <ion-list>
+        <ion-card
+          v-for="workspace in workspaces"
+          :key="workspace.id"
+          :disabled="user.isRevoked()"
+          class="workspace-card"
+        >
+          <ion-card-content>
+            <ion-avatar class="card-content-icons">
+              <ion-icon
+                class="card-content-icons__item"
+                :icon="business"
+              />
+              <ion-icon
+                class="cloud-overlay"
+                :class="workspace.availableOffline ? 'cloud-overlay-ok' : 'cloud-overlay-ko'"
+                :icon="workspace.availableOffline ? cloudDone : cloudOffline"
+              />
+              <ion-text class="text3">
+                {{ workspace.name }}
+              </ion-text>
+              <workspace-tag-role :role="workspace.role" />
+            </ion-avatar>
+          </ion-card-content>
+        </ion-card>
+      </ion-list>
+      <ion-button
+        class="close-button"
+        @click="cancel"
+      >
+        {{ $t('UsersPage.UserDetailsModal.actions.close') }}
+      </ion-button>
+    </ms-modal>
+  </ion-page>
+</template>
+
+<script setup lang="ts">
+import { MsModal } from '@/components/core';
+import { MsModalResult } from '@/components/core/ms-modal/types';
+import {
+  IonChip,
+  IonAvatar,
+  IonButton,
+  IonList,
+  IonPage,
+  IonText,
+  modalController,
+  IonCard,
+  IonCardContent,
+  IonIcon,
+  IonLabel,
+} from '@ionic/vue';
+import { business, cloudDone, cloudOffline, ellipse } from 'ionicons/icons';
+import { UserInfo } from '@/parsec';
+import { FormattersKey, Formatters } from '@/common/injectionKeys';
+import { defineProps, inject } from 'vue';
+import { WorkspaceRole } from '@/parsec';
+import WorkspaceTagRole from '@/components/workspaces/WorkspaceTagRole.vue';
+
+// eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+const { timeSince } = inject(FormattersKey)! as Formatters;
+const workspaces = [
+  {
+    id: 'fake1',
+    name: 'Workspace1',
+    role: WorkspaceRole.Owner,
+    availableOffline: true,
+  },
+  {
+    id: 'fake2',
+    name: 'Workspace2',
+    role: WorkspaceRole.Contributor,
+    availableOffline: false,
+  },
+];
+
+defineProps<{
+  user: UserInfo;
+}>();
+
+function cancel(): Promise<boolean> {
+  return modalController.dismiss(null, MsModalResult.Cancel);
+}
+</script>
+
+<style lang="scss" scoped>
+.text1 {
+  font-size: 0.9rem;
```

Comment:
You don't need the font specifications because we have a predefined class that handles each text for `line-height`, `font-size`, `font-weight` and `letter-spacing`. 
On Figma, when you click on a text you will see something like "Body/Small" so the class is `class="body-sm"`(you can find all class names in `theme`/`variables`/`fonts`)